azero/contracts/most/lib.rs

```diff
@@ -747,6 +755,22 @@ pub mod most {
             Ok(self.data()?.is_halted)
         }
 
+        /// Returns the status of a given cross-chain transfer request
+        #[ink(message)]
+        pub fn request_status(&self, hashed_request: HashedRequest) -> RequestStatus {
```

Comment:
Question: I like this version much better, but you also removed the `has_signed` method - was that not needed for anything?

Reply:
It could be used eg by relayer, but I don't think it is useful in the front-end